packages/devtools_shared/lib/src/deeplink/deeplink_manager.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:shelf/shelf.dart' as shelf;
+
+import '../server/server_api.dart';
+
+class DeeplinkManager {
+  static final _buildVariantJsonRegex = RegExp(r'(\[.*\])');
+
+  @visibleForTesting
+  Future<ProcessResult> runProcess(String executable, List<String> arguments) {
+    return Process.run(executable, arguments);
+  }
+
+  Future<shelf.Response> getBuildVariants({
+    required String rootPath,
+    required ServerApi api,
+  }) async {
+    final ProcessResult result = await runProcess(
+      'flutter',
```

Comment:
I think this is dangerous, is there a safer way to run flutter binary?

Reply:
What makes this dangerous? @christopherfujino do you have an opinion here?

Reply:
This assume the flutter is in the path environment variable.